SwiftyJSONAccelerator/Base Files/ObjectMapperTemplate.txt

```diff
@@ -0,0 +1,15 @@
+        /**
```

Comment:
If you remove the tabs, the alignment is no longer correct. There's an issue with indentations of the first line of many methods, however removing all of the tabs in the template files won't resolve it. It'll actually make the problem worse (which is why they were added).

I'll see what kind of cleanup I can do to fix the alignment of the few lines that are off.